src/client/index.tsx

```diff
@@ -1,42 +1,31 @@
 import "raf/polyfill";
 
 import App from "@shared/App";
+import rootReducer from "@shared/store/rootReducer";
+import rootSaga from "@shared/store/rootSaga";
 import * as OfflinePluginRuntime from "offline-plugin/runtime";
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import { Provider } from "react-redux";
 import { BrowserRouter } from "react-router-dom";
-import { Action, createStore } from "redux";
+import { applyMiddleware, compose, createStore } from "redux";
+import createSagaMiddleware from "redux-saga";
 import "../shared/styles/core/core.scss";
 
 /**
  * Define root mounting point
  */
 const root = document.getElementById("root");
 
-/**
- * Create root reducer
- * @param state
- * @param action
- *
- * @todo Move this to a new location
- */
-const rootReducer = (state: string = "asdf", action: Action) => {
-    switch (action.type) {
-        default:
-            return state;
-    }
-};
-
 /**
  * Setup redux dev tool
  */
 const reduxDevToolExtension = () => {
     return ((w: Window) => {
-        if (w && w.__REDUX_DEVTOOLS_EXTENSION__) {
-            return w.__REDUX_DEVTOOLS_EXTENSION__();
+        if (w && w.devToolsExtension) {
+            return w.devToolsExtension();
         }
-        return undefined;
+        return () => undefined;
```

Comment:
Typescript doesn't seem to like ()=>{} (empty block) so I read you can do this instead, let me know if that's not the right way to do it.

Reply:
Why does it need to return a anonymous function that then returns `undefined`? Isn't returning `undefined` the same result?

Reply:
@jimmynotjim From what I can tell, the return value of this `reduxDevToolExtension` is being used as a [redux store enhancer](https://redux.js.org/glossary#store-enhancer) down below on line 40, so it must always return a function.